src/main/java/robot/subsystems/hatchIntake/groundIntake.java

```diff
@@ -19,6 +19,8 @@
     // here. Call these from Commands.
     private final DoubleSolenoid grountIntake = new DoubleSolenoid(Ports.groundForward, Ports.groundReverse);
     private final AnalogInput proxSensor = new AnalogInput(Ports.prox);
+    private final DoubleSolenoid flowerExtension = new DoubleSolenoid(Ports.flowerExtensionForward, Ports.flowerExtensionReverse);
```

Comment:
I understand we called it _Flower_, but I think we should give it a more professional name. Maybe _Gripper_ is a better name.